packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.component.js

```diff
@@ -122,4 +132,9 @@ SimpleTextKeyValue.propTypes = {
 	value: PropTypes.object,
 	separator: PropTypes.string,
 	style: PropTypes.object,
+	displayTypes: PropTypes.bool.isRequired,
```

Comment:
isRequired can be removed as there is a defaultProps for it

Reply:
```suggestion
	displayTypes: PropTypes.bool,
```